musicrs/util/audioTrimmer.py

```diff
@@ -0,0 +1,17 @@
+from pydub import AudioSegment
+import sys
+sys.path.append('/usr/bin/ffmpeg')
+
+def audioTrimmer(file_name,startMin,startSec,endMin,endSec):
+  files_path = 'mp3/'
```

Comment:
files path should be in parameter.. Utils should be as generic as possible. Its only task is to trim the audio the necessary information should be given by the service file which uses this util